packages/rum/src/domain/segmentCollection/segmentCollection.spec.ts

```diff
@@ -124,27 +117,11 @@ describe('startSegmentCollection', () => {
     })
 
     it('flushes segment when the page become hidden', () => {
-      setPageVisibility('hidden')
-      const { addRecord, eventEmitter, sendCurrentSegment, worker, sendSpy } = startSegmentCollection(CONTEXT)
-      addRecord(RECORD)
-      eventEmitter.dispatchEvent(createNewEvent(DOM_EVENT.VISIBILITY_CHANGE))
-
-      worker.processAllMessages()
-      expect(sendSpy).toHaveBeenCalledTimes(1)
+      const { lifeCycle, sendCurrentSegment } = startSegmentCollection(CONTEXT)
+      lifeCycle.notify(LifeCycleEventType.PAGE_EXITED, { isUnloading: false })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** update test name to something like `flushes segment on page exit`‏

Reply:
In the previous PR you [suggested](https://github.com/DataDog/browser-sdk/pull/1788#discussion_r1004680648) to keep a test for `{ isUnloading: false }`. Since there is also a test for `{ isUnloading: true }`, we should have more specific names than `flushes segment on page exit`.


What do you think about:
* `flushes segment on page exit because it gets hidden`
* `flushes segment on page exit because it is unloading`

Reply:
sounds good to me!